### PR TITLE
Added the missing set namespace shell script + Fix Deploys

### PR DESCRIPTION
**Pull request by CCP-Red-Dragon**
The namespace shell script seemed to be missing from my previous PR - so have added it here with some additional fixes.

Have also added a fix for deploying to garnet as it wasn't properly reading the environment variables.

### CODE REVIEW

File: smart-gate/shell-scripts/SetNamespace.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+#Colours
+GREEN="\033[32m"
+YELLOW="\033[33m"
+RESET="\033[0m"
+
+function validate_input(){
+    local INPUT=""
+    read -p "Please insert your $1: " INPUT
+    local MIN_LENGTH="$2"
```
Comment (0xxlegolas) [inline smart-gate/shell-scripts/SetNamespace.sh:14]: can we also add a max length as namespace is 14 bytes and do a check

Comment (CCP-Red-Dragon) [reply smart-gate/shell-scripts/SetNamespace.sh:14]: Good idea, thank you :) Have added

**Comment by CCP-Red-Dragon** [comment]
Have added a fix for deploying to garnet - as it wasn't properly reading the world address from env variables.

**Comment by 0xxlegolas** [comment]
Can we add some docs for this feature ?

**Comment by 0xxlegolas** [comment]
Looks like it does not work in MAC
image

**Comment by 0xxlegolas** [comment]
Still messes up when the input is given second time 
image

File: smart-turret/packages/contracts/.env [hunk 2]
```diff
@@ -12,7 +12,7 @@ PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
 WORLD_ADDRESS=0x8a791620dd6260079bf849dc5567adc3f2fdc318 #Local World Address
 
 #Local RPC
-RPC_URL=http://127.0.0.1:8545 #Forked Anvil Local RPC URL
+RPC_URL="http://127.0.0.1:8545" #Forked Anvil Local RPC Url
```
Comment (0xxlegolas) [inline smart-turret/packages/contracts/.env:15]: does it have to be quoted ?

**Review by 0xxlegolas — APPROVED** [review]

File: smart-turret/packages/contracts/.env (hunk 2 above)
Comment (CCP-Red-Dragon) [reply smart-turret/packages/contracts/.env:15]: It does - this seems to be a MacOS issue which I will solve in a further PR :)

**Pull request merged**